### PR TITLE
[brynet] upgrade to 1.11.2

### PR DESCRIPTION
**Pull request by IronsDu**
**Describe the pull request**

- #### What does your PR fix?
No

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
All

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
Yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

**Comment by IronsDu** [comment]
@JackBoosY Can you help me for review this PR's failed?

File: ports/brynet/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "brynet",
-  "version-string": "1.11.1",
+  "version-string": "1.11.2",
```
Comment (JackBoosY) [inline ports/brynet/vcpkg.json:3]:
```suggestion
  "version": "1.11.2",
```

File: versions/b-/brynet.json [hunk 2]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "a107839fbb9e66cd6c664bc4bf0b7cc278d1c252",
```
Comment (JackBoosY) [inline versions/b-/brynet.json:4]:
```suggestion
      "git-tree": "ef613c0010e513fb902ed5a19b3874dc7d8f696b",
```

File: ports/brynet/vcpkg.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "brynet",
-  "version-string": "1.11.1",
+  "version": "1.11.2",
   "port-version": 1,
```
Comment (JackBoosY) [inline ports/brynet/vcpkg.json:4]:
```suggestion
```

File: versions/b-/brynet.json [hunk 4]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "ef613c0010e513fb902ed5a19b3874dc7d8f696b",
+      "version": "1.11.2",
+      "port-version": 1
```
Comment (JackBoosY) [inline versions/b-/brynet.json:6]:
```suggestion
      "git-tree": "3a4cea82b4e937b478bd400d4a975dde33b3f741",
      "version": "1.11.2",
      "port-version": 0
```

File: versions/baseline.json [hunk 5]
```diff
@@ -1153,7 +1153,7 @@
       "port-version": 0
     },
     "brynet": {
-      "baseline": "1.11.1",
+      "baseline": "1.11.2",
       "port-version": 1
```
Comment (JackBoosY) [inline versions/baseline.json:1157]:
```suggestion
      "port-version": 0
```

**Review by JackBoosY — APPROVED** [review]

**Comment by strega-nil-ms** [comment]
Thanks!

**Pull request merged**